backend/server/schedule_get.py

```diff
@@ -0,0 +1,47 @@
+from flask import Blueprint, jsonify
+from tools.get_slots_by_filter import get_slots_by_filter
+from tools.get_slots_in_week import get_slots_in_week
+from tools.get_slots_in_day import get_slots_in_day
+from tools.get_slots_in_week_by_filter import get_slots_in_week_by_filter
+from tools.get_slots_in_day_by_filter import get_slots_in_day_by_filter
+
+
+schedule_get = Blueprint('schedule_get', __name__)
+
+
+@schedule_get.route("/schedule/filter=status=<filter>", methods=['GET'])
```

Comment:
looks nice.
I would add two things:
1) The existing tests are great. I would add some tests of the enpoints using `flask` test client so that we can check integration between flask and functions like `get_slots_by_filter`. Instead of calling some implementation function as we do in our current tests, those integration tests would make a HTTP call to `/schedule` endpoint. 
2) API specs and maybe tests for incorrect input. I guess an exception will be thrown at some point if we specify a wrong date for example 2021-42-42. It would be great if we handle that exception nicely in our API. Flask has a functionalyty to set handlers for exceptions. In that case we can reply to the caller with a valid json with error description and something like `bad Request` status

Reply:
in the new issues:
* https://github.com/meetingbook/meetingbook/issues/82
* https://github.com/meetingbook/meetingbook/issues/83